executor/executor.go

```diff
@@ -134,6 +147,18 @@ func newBaseExecutor(ctx sessionctx.Context, schema *expression.Schema, id strin
 	return e
 }
 
+type baseNoResultExecutor struct {
```

Comment:
What is this?

Reply:
this is make a sub-executor to override `newChunk()` method to make updateExec/insertExec new a void chunk~

Reply:
Why we need this?